app/views/moderations/_suspend_user_modal.html.erb

```diff
@@ -0,0 +1,63 @@
+<div id="suspend-unsuspend-modal" class="hidden">
```

Comment:
This file handles the modal contents for both the Suspend and Unsuspend User flows.

Reply:
(nitpick / not blocking) Should the file maybe be called something like `_suspend_unsuspend_user_modals`? or even just `_suspend_user_modals` just to be clear there's more than one modal in here?